create_infotexts.py

```diff
@@ -154,9 +154,11 @@ def generate_infobox_template(item, img, places_mapping):
     infobox += "| depicted place     = " + img.data["depicted_place"] + "\n"
 
     infobox += "| date               = "
-    if not item["Fotodatum"] == "":
-        infobox += str(item["Fotodatum"])
-    infobox += "\n"
+    if item["Fotodatum"]:
```

Comment:
This wont work since if `item["Fotodatum"] = ""` this will give false.

instead use:
```python
if item["Fotodatum"] is not None:
    if item["Fotodatum"] == "1927-1931" or item["Fotodatum"] == "":
        infobox += "{{Between|1927|1931}}"
    else:
        infobox += str(item["Fotodatum"])
infobox += "\n"
```

Note that the last `+= "\n"` need to be outside of the if so that it is applied even if Fotofatum is None.